scripts/game.js

```diff
@@ -1,19 +1,92 @@
 //Game functions
-let rabbitAppearTime = []; //Empty array that we can fill with the times each rabbit appeared
-let rabbitClickedTimes = []; //Empty array we can fill with the time each rabbit was clicked
+let time1 = new Date(2021, 9, 13, 22, 10, 30, 100); console.log(time1);
+let time2 = new Date(2021, 9, 13, 22, 10, 31, 200);
+let time3 = new Date(2021, 9, 13, 22, 10, 31, 900);
+let time4 = new Date(2021, 9, 13, 22, 10, 32, 700);
 
-let numRabbits = 0;
-let rabbitPenalty = 0;
-let misclickPenalty = 0;
+let rabbitAppearTime = [time1, time2, time3, time4]; //Empty array that we can fill with the times each rabbit appeared
+let rabbitClickedTimes = [ , time3, , ]; //Empty array we can fill with the time each rabbit was clicked
+let finalTime = 0;
+let today = new Date();
+let date = today.getDate() + "/" + (today.getMonth()+1) + "/" + today.getFullYear();
+let avReactSpeed = 0;
+let difficulty = "easy";
+// let numRabbits = 0;
+// Sum of not clicked on the rabbit
+let rabbitPenalty = 2;
+// Sum of clicking wrong
+let misclickPenalty = 1;
 
+// let difficulty = "easy";
+// let starttimeGamesession, endtimeGamesession;
+timerFunction();
 function timerFunction () {
-    //Hard one. Will need to save the time when the game is started into a variable
+    finalTime = 0;
+    for(let i = 0; i < rabbitAppearTime.length; i++){
+        if(rabbitClickedTimes[i] != null){
+            finalTime += (rabbitClickedTimes[i].getTime() - rabbitAppearTime[i].getTime());
+        }
+    }
+    //Depending on how much we want the penaltys to be depending on difficulty?*************************
+    finalTime += (rabbitPenalty * penaltyNumber()); // example with "easy" 2000 milliseconds 
+    finalTime += (misclickPenalty * penaltyNumber()); // should we set this different??
```

Comment:
I think we discussed having a lower penalty for a misclick than missing a rabbit. I think that would make hard mode more balanced.